db/controllers/errors.controller.js

```diff
@@ -1,7 +1,14 @@
 exports.handleCustomErrors = (err, req, res, next) => {
     if (err.status) {
         res.status(err.status).send({ msg: err.message });
-    } else {
+    }
+    if (err.code === '22P02') {
+        res.status(400).send({ msg: 'bad request!'})
+    }
+    if (err.code === '23503') {
+        res.status(404).send({ msg: 'not found' })
+    }
```

Comment:
This should be a 400 as it is an SQL error. If a user doesn't input all the information required, this is user error and is  a bad request